test/native/setup.js

```diff
@@ -132,6 +132,19 @@ jest.mock( 'react-native-safe-area', () => {
 	};
 } );
 
+jest.mock( 'react-native-safe-area-context', () => {
+	const inset = { top: 0, right: 0, bottom: 0, left: 0 };
+	return {
+		SafeAreaProvider: jest
+			.fn()
+			.mockImplementation( ( { children } ) => children ),
+		SafeAreaConsumer: jest
+			.fn()
+			.mockImplementation( ( { children } ) => children( inset ) ),
+		useSafeAreaInsets: jest.fn().mockImplementation( () => inset ),
+	};
+} );
```

Comment:
I noticed that in a newer version of the package, they already provide a jest mock 😅:
https://github.com/th3rdwave/react-native-safe-area-context/releases/tag/v3.3.0

This mock should be fine but we might consider in the future replace it with the one provided by the new version.

Reply:
Yeah I saw that it is included in a newer version but I didn't want to go that route during this project 😄 I'll add a comment as a reference so we can remove it once we update.

Reply:
Yeah makes totally sense, upgrading the library might introduce new bugs, so I think it would be better to do it in a separate PR in the future 👍 .